content/getting-started/setup/upgrades/index.md

```diff
@@ -8,4 +8,63 @@ dnnversion: 09.04.03
 # DNN Platform Upgrades
 Following are the basic steps for upgrading **DNN Platform**.
 
-Please click the **Improve this Doc** button above to help us improve this page.
\ No newline at end of file
+> [!NOTE]
+> Every upgrade scenario is unique in some shape, form or fashion. This document is by no means intended to address every scenario. Some upgrades can be quite complex and it is virtually impossible to cover every scenario. This document is intended to focus only on the upgrade basics.
+
+## Step 1 - Prepare
+Before starting the upgrade process, it is recommended to have a very clear understanding of all upgrade requirements and expectations. Here are some tips to help prepare.
+
+* Know the current version of DNN Platform.
+
+* Ensure **AutoUpgrade** setting is "False". This setting can be found within **AppSettings** section of the `web.config` file in the root. Setting this to "False" will prevent the upgrade process from triggering by normal site visitors. 
```

Comment:
...from being triggered by normal site visitors.

Reply:
Okay, I'll submit a new PR to change this.  This PR has already been reviewed, approved and merged by @donker.  ;-)

Reply:
@donker was not cut out to be an editor ;)